src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/CounterCreationData.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Diagnostics
+{
+    /// <summary>
+    ///     A class defining the counter type, name and help string for a custom counter.
+    /// </summary>
+    public class CounterCreationData
```

Comment:
I believe this should be a struct.

| enum (4 bytes) |
| string (4 bytes) |
| string (4 bytes) |

This sums up to 12 bytes which is smaller than 16 bytes which is the recommended maximum for a struct.

@danmosemsft is changing the type from class to struct considered as a breaking change?

Reply:
I'm not sure it can be a struct since its public default constructor initializes the fields to non default values? 

But anyway this falls into the category of refactoring, which I strongly recommend deferring until there's tests.

Reply:
It does make sense to fix style and remove dead code though. I think that is safe enough.

Reply:
You are right, I didn't notice that the fields values shouldn't be zeroed out with the default ctor.